### DISCUSSION TITLE
Running AutoUpdate on a computer that has Windows Scripting Host disabled

### DISCUSSION THREAD

**Discussion by thomasjsweet** [Romanitho/Winget-AutoUpdate#376] [Q&A]
Hello
All our computers have this reg key set by Intune
`Set-ItemProperty -Path 'HKLM:\Software\Microsoft\Windows Script Host\Settings\' -Name  Enabled -Value 0 -Force`
As part of our security baseline.

When Winget AutoUpdate runs, we get a dialog box stating "Windows scripting host is disabled on this system" for each update that Winget updates. The updates work but end users keep reporting bugs to us about the message.

Any recommendations?

**Comment by Romanitho** [comment]
https://github.com/Romanitho/Winget-AutoUpdate/blob/main/Winget-AutoUpdate/functions/Update-App.ps1
this is the function called when an update is installed. Never tried with WSH disabled. I will try when I have some time.
